Documentation/dev-guide/limit.md

```diff
@@ -2,7 +2,7 @@
 
 ## Request size limit
 
-etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. For the time being, etcd guarantees to support RPC requests with up to 1MB of data. In the future, the size limit may be loosened or made configurable.
+etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. As of v3.2.12, etcd guarantees to support client RPC requests of 2 MiB, by default. The client size limit is configurable via `clientv3.Config.MaxCallSendMsgSize`, with corresponding "--max-request-bytes" flag for etcd server.
```

Comment:
@gyuho a small observation,  "--max-request-bytes" should change to `--max-request-bytes`. Thanks!